README.md

```diff
@@ -1,3 +1,17 @@
+---
+
+This library, ADAL for .NET, will no longer receive new feature improvements. Instead, use the new library
+[MSAL for .NET](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet).
+
+* If you are starting a new project, you can get started with the
+  [MSAL for .NET docs](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki)
```

Comment:
should we be linking to ms docs and not the wiki? @jmprieur?

Reply:
Can you guys help me understand which docs page you want me to link to? When I scan the docs, I don't see any obvious candidates:
- [There's the general learn about MSAL page](https://docs.microsoft.com/azure/active-directory/develop/msal-overview)
- [There's the overall page for all MSAL libraries](https://docs.microsoft.com/azure/active-directory/develop/reference-v2-libraries)
- [There's the raw API reference list](https://docs.microsoft.com/en-us/dotnet/api/microsoft.identity.client?view=azure-dotnet-preview)

It doesn't seem like there's a per-platform place to discuss resources in the docs, as opposed to the Github? Or am I just looking in the wrong place?

Reply:
There's not an obvious place in the docs to link to, so I will leave it for now and when we migrate more of the wiki over to the docs, these links can be updated.